### ISSUE TITLE
Can i get the merchantSig & orderData like merchantRef

### ISSUE THREAD

**Issue by ravindranpandu** [hekike/adyen-node#10]
Hi,

I am using `adyen-node` for a while, its really cool but i am just wondering is there a way to get the merchantSig and orderData instead of getting it appended in the url?
```
hppPayment.generateRequest(function (error, paymentUrl, merchantReference) {
     // can i get the url params here as a jsonData?
});
```
Thanks

**Comment by onchainguy-btc** [comment]
You could use https://nodejs.org/docs/latest/api/url.html#url_urlobject_query

**Comment by ravindranpandu** [comment]
@seb0zz thanks for the reply, that's what i am doing currently, was expecting a cleaner way to achieve it.

**Comment by onchainguy-btc** [comment]
Should be easy to implement if you take a look at [HPP.js](https://github.com/hekike/adyen-node/blob/master/lib/HPP.js) If you could provide a PR and add a test I can merge it.

**Comment by onchainguy-btc** [comment]
Closed with #11
